layer.md

```diff
@@ -66,23 +112,33 @@ Modified:   /bin/my-app-tools
 Deleted:    /etc/my-app-config
 ```
 
-A Tar Archive is then created which contains *only* this changeset:
+This reflects the removal of `/etc/my-app-config` and creation of a file and directory at `/etc/my-app.d/default.cfg`.
+`/bin/my-app-tools` has also been replaced with an updated version.
 
-- Added and modified files and directories in their entirety
-- Deleted files or directory marked with a whiteout file
+### Representing Changes
 
-A whiteout file is an empty file that prefixes the deleted paths basename `.wh.`.
-When a whiteout is found in the upper changeset of a filesystem, any matching name in the lower changeset is ignored, and the whiteout itself is also hidden.
-As files prefixed with `.wh.` are special whiteout tombstones it is not possible to create a filesystem which has a file or directory with a name beginning with `.wh.`.
+A [tar archive][tar-archive] is then created which contains *only* this changeset:
 
-The resulting Tar archive for `f60c56784b83` has the following entries:
+- Added and modified files and directories in their entirety
+- Deleted files or directory marked with a [whiteout file](#whiteouts)
+
+The resulting Tar archive for `rootfs-c9d-v1.s1` has the following entries:
 
 ```
-/etc/my-app.d/default.cfg
-/bin/my-app-tools
-/etc/.wh.my-app-config
+./etc/my-app.d/
+./etc/my-app.d/default.cfg
+./bin/my-app-tools
+./etc/.wh.my-app-config
 ```
 
+## Applying
+
+## Whiteouts
+
+A whiteout file is an empty file that prefixes the deleted paths basename with `.wh.`.
```

Comment:
fixed

On Tue, Sep 6, 2016 at 2:06 PM Jonathan Boulle <notifications@github.com>
wrote:

> In layer.md
> <https://github.com/opencontainers/image-spec/pull/255#discussion_r77687720>
> :
>
> >  ```
> >
> > +## Applying
> > +
> > +## Whiteouts
> > +
> > +A whiteout file is an empty file that prefixes the deleted paths basename with `.wh.`.
>
> A whiteout file an empty file with a special filename that signifies a
> path should be deleted. A whiteout filename consists of the prefix .wh.
> plus the basename of the path to be deleted.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/opencontainers/image-spec/pull/255/files/c3758476b86455a2dab2c7522f3e9aeedf3347e3#r77687720>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAEF6ap9JAMmvRwdo0VlJKlGLw-3rdgrks5qnauFgaJpZM4J0hox>
> .
>